pyfolio/tests/test_tears.py

```diff
@@ -26,7 +26,7 @@ class PositionsTestCase(TestCase):
     test_returns = read_csv(
         gzip.open(
             __location__ + '/test_data/test_returns.csv.gz'),
-        index_col=0, parse_dates=True)
+        index_col=0, parse_dates=True).iloc[4:]
```

Comment:
those were all 0's. it was part of me tracking down a problem but I think the problem was somewhere else so we can try removing this again.